docs/sources/trackers/javascript-trackers/web-tracker/cross-domain-tracking/index.md

```diff
@@ -4,23 +4,351 @@ date: "2022-08-30"
+
+### Decorate some links
+
+For example, this function would only decorate those links whose destination is `http://acme.de/` or whose HTML ID is `crossDomainLink`:
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="js" label="JavaScript (tag)" default>
+
+```javascript
+snowplow('newTracker', 'namespace', '{{collector_url_here}}', {
+  appId: 'my-app-id',
+  appVersion: '0.1.0',
+  cookieSameSite: 'Lax',
+  crossDomainLinker: function (linkElement) {
+    return (linkElement.href === 'http://acme.de' || linkElement.id === 'crossDomainLink');
+  }
+});
+```
+
+  </TabItem>
+  <TabItem value="browser" label="Browser (npm)">
+
+```javascript
+newTracker('namespace', '{{collector_url_here}}', {
+  appId: 'my-app-id',
+  appVersion: '0.1.0',
+  cookieSameSite: 'Lax',
+  crossDomainLinker: function (linkElement) {
+    return (linkElement.href === 'http://acme.de' || linkElement.id === 'crossDomainLink');
+  }
+});
+```
+
+  </TabItem>
+</Tabs>
+
+If you want to decorate every link to the domain `github.com`:
+
+```javascript
+{
+  crossDomainLinker: function (linkElement) {
+    return /^https:\/\/github\.com/.test(linkElement.href);
+  }
+}
+```
+
+You can also call the `crossDomainLinker` function directly:
+
 <Tabs groupId="platform" queryString>
   <TabItem value="js" label="JavaScript (tag)" default>
 
-<CrossDomain lang="javascript" />
+```javascript
+snowplow('crossDomainLinker', function(linkElement) {
+  return (linkElement.href === 'http://acme.de' || linkElement.id === 'crossDomainLink');
+});
+```
 
   </TabItem>
   <TabItem value="browser" label="Browser (npm)">
 
-<CrossDomain lang="browser" />
+```javascript
+crossDomainLinker(function(linkElement) {
+  return (linkElement.href === 'http://acme.de' || linkElement.id === 'crossDomainLink');
+});
+```
 
   </TabItem>
 </Tabs>
+
+
+### Decorate all links
+
+To decorate every link, regardless of its destination:
+
+```javascript
+{
+  crossDomainLinker: function (linkElement) {
+    return true;
+  }
+}
+```
+
+This will decorate "links" that are actually just JavaScript actions (with an `href` of `"javascript:void(0)"`), or links to email addresses or telephone numbers (`mailto:` or `tel:` schemes). To exclude these links, check them explicitly:
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="js" label="JavaScript (tag)" default>
+
+```javascript
+snowplow('crossDomainLinker', function(linkElement) {
+  return linkElement.href.indexOf('javascript:') < 0;
+});
+```
+
+  </TabItem>
+  <TabItem value="browser" label="Browser (npm)">
+
+```javascript
+crossDomainLinker(function(linkElement) {
+  return linkElement.href.indexOf('javascript:') < 0;
+});
+```
+
+  </TabItem>
+</Tabs>
+
+Alternatively, only count links that parse as web URLs by checking the link's [`hostname`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement/hostname). This should automatically exclude links that don't lead simply to other web pages.
```

Comment:
> This should automatically exclude links that don't lead simply to other web pages.

I'm not sure what this means?

Probably more useful to check the hostname against an array of domains or something.